specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -16,119 +17,137 @@ enum TextCategory {
 }
 
 #suppress "@azure-tools/typespec-azure-core/documentation-required" "MUST fix in next update"
-@doc("Image analyze category")
+@doc("Image analyze category.")
 enum ImageCategory {
   Hate,
   SelfHarm,
   Sexual,
   Violence,
 }
 
-@doc("The analysis request of the text.")
+@doc("The type of text analysis output.")
+enum AnalyzeTextOutputType {
```

Comment:
"Kind" is preferred over "Type" since "Type" can mean many different things in SDK languages.